.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: []
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting this issue.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
```

```suggestion
  - type: input
```
Since this will probably be a single line response, input might make the page prettier!